src/vma/proto/dst_entry_tcp.cpp

```diff
@@ -171,17 +159,11 @@ ssize_t dst_entry_tcp::fast_send(const struct iovec* p_iov, const ssize_t sz_iov
 		send_ring_buffer(m_id, m_p_send_wqe, b_blocked, is_dummy);
 	}
 
-#ifndef __COVERITY__
-        struct tcphdr* p_tcp_h = (struct tcphdr*)(((uint8_t*)(&(p_pkt->hdr.m_ip_hdr))+sizeof(p_pkt->hdr.m_ip_hdr)));
-        NOT_IN_USE(p_tcp_h); /* to supress warning in case VMA_OPTIMIZE_LOG */
-        dst_tcp_logfunc("Tx TCP segment info: src_port=%d, dst_port=%d, flags='%s%s%s%s%s%s' seq=%u, ack=%u, win=%u, payload_sz=%u",
```

Comment:
Better to keep the debug prints
I see they area already after the send operation. 
unless you do full debug prints cleanup for a RELEASE mode... very hard to debug cases later on